### PR TITLE
#3862 [Manager] redesign event logs structure

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3862 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/api/utils/apiUtils.js [hunk 1]
```diff
@@ -24,10 +24,8 @@ import {
 import axios from 'axios';
 import { has, isEmpty, isArray } from 'lodash';
 
-export const getKey = params => {
-  const name = params.name || params.hostname || undefined;
-  const group = params.group || undefined;
-  return JSON.stringify({ group, name });
```
Comment (konekoya) [inline ohara-manager/client/src/api/utils/apiUtils.js]: Good one, why we need to use `JSON.stringify()` in the first place 🤔

Comment (saivirtue) [reply ohara-manager/client/src/api/utils/apiUtils.js]: in order to show the format `{group: xxx, name: yyy}` :)

File: ohara-manager/client/src/api/objectApi.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const create = async (params = {}) => {
 
   const result = responseUtil(res, objBody);
   result.title =
-    `Create object ${getKey(params)} ` +
+    `Create object ${getName(params)} ` +
```
Comment (konekoya) [inline ohara-manager/client/src/api/objectApi.js]: Are we showing this for the users after an action is complete? If so, we should change all these message to start with a past tense sentence... like: 
_Failed to create object abc_
_Successfully created object abc_
_Failed to update object abc_
_Successfully updated object abc_

These read better IMO and sorry these should be caught on the initial commit not this one 😅

Comment (eechih) [reply ohara-manager/client/src/api/objectApi.js]: Good suggestion~  

> sorry these should be caught on the initial commit not this one 😅

Not too late

File: ohara-manager/client/src/api/utils/apiUtils.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/api/utils/apiUtils.js]: Oh, okay. So plain won't work?  like `{ ${group} ${name} }`

File: ohara-manager/client/src/utils/hooks.js [hunk 3]
```diff
@@ -64,19 +65,41 @@ export const useLocalStorage = (key, initialValue) => {
...
-        value instanceof Function ? value(storedValue) : value;
-      // Save state
-      setStoredValue(valueToStore);
-      // Save to local storage
-      window.localStorage.setItem(key, JSON.stringify(valueToStore));
-    } catch (error) {
-      // ignore the error for now
-    }
-  };
+  const setValue = useCallback(
```
Comment (konekoya) [inline ohara-manager/client/src/utils/hooks.js]: 👍 Thanks for fixing this @eechih

File: ohara-manager/client/src/api/utils/apiUtils.js (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/client/src/api/utils/apiUtils.js]: it will display like `{ {object} {object} }` 😅

Comment (konekoya) [reply ohara-manager/client/src/api/utils/apiUtils.js]: That's why, thanks for the detailed explanation, Sam :)

**Comment by saivirtue** [comment]
BTW, it seems #3868 has some conflict changes with this PR...I will wait for this PR merged first.

**Pull request merged**